src/crypto/CHIPCryptoPAL.h

```diff
@@ -1110,11 +1123,29 @@ class Spake2p_P256_SHA256_HKDF_HMAC : public Spake2p
     Spake2pOpaqueContext mSpake2pContext;
 };
 
-/** @brief Clears the first `len` bytes of memory area `buf`.
- * @param buf Pointer to a memory buffer holding secret data that should be cleared.
+/**
+ * @brief Compute the compressed fabric identifier used for operational discovery service
+ *        records from a Node's root public key and Fabric ID. On success, out_compressed_fabric_id
+ *        will have a size of exactly 8.
+ *
+ * Errors are:
+ *   - CHIP_ERROR_INVALID_ARGUMENT if root_public_key is invalid
+ *   - CHIP_ERROR_BUFFER_TOO_SMALL if out_compressed_fabric_id is too small for serialization
+ *   - CHIP_ERROR_INTERNAL on any unexpected crypto errors.
+ *
+ * @param[in] root_public_key The root public key associated with the node's fabric
+ * @param[in] fabric_id The fabric ID associated with the node's fabric
+ * @param[out] out_compressed_fabric_id Span where output will be written. Its size must be >= 8.
```

Comment:
size must be >= sizeof(uint64_t).

Reply:
Well, really it's 8 bytes. The compressed fabric identifier is considered an octet string in the spec.

I will create a constant so I don't use `sizeof(uint64_t)`.